fw/Core/Hitcon/Logic/Display/display.h

```diff
@@ -1,6 +1,7 @@
 #ifndef DISPLAY_H
 #define DISPLAY_H
 
+#include <Logic/Display/font.h>
 #include <stdint.h>
 
 /**
```

Comment:
Since the underlying memory layout is modified, I would suggest updating this comment to help others understand how `display_buf_t` stores the display buffer.

Reply:
```
/*
 * The underlying buffer is a 2D array of uint8_t like this:
 *
 * buf[0][0] buf[0][1] ... buf[0][DISPLAY_WIDTH - 1]
 * buf[1][0] buf[1][1] ... buf[1][DISPLAY_WIDTH - 1]
 * ...
 * buf[DISPLAY_HEIGHT - 1][0] ... buf[DISPLAY_HEIGHT - 1][DISPLAY_WIDTH - 1]
 */
```